HERE_NOTICE

```diff
@@ -0,0 +1,720 @@
+Licenses and Copyright Notices:
+================================================================================
+
+This software depends on external packages and source code.
+The applicable license information is listed below:
+
+----
+(c) 2014-2023 Denis Pushkarev
+Copyright (c) 2017 $YEAR HERE Europe B.V.
```

Comment:
shouldn't we change YEAR value to 2024 in build.gradle.kts now?

Reply:
addressed in both v2 and v3 branches